services/src/handlers/wms.rs

```diff
@@ -447,6 +445,8 @@ mod tests {
         check_allowed_http_methods(get_capabilities_test_helper, &[Method::GET]).await;
     }
 
+    // The result should be similar to the GDAL output of this command: gdalwarp -tr 1 1 -r near -srcnodata 0 -dstnodata 0  MOD13A2_M_NDVI_2014-01-01.TIFF MOD13A2_M_NDVI_2014-01-01_360_180_near_0.TIFF
+    /*
     #[tokio::test]
     async fn png_from_stream_non_full() {
```

Comment:
why is this test commented out?

Reply:
i told you about that in slack. the test is broken. With the old implementation it is fine. However, the result does not match the gdal result. The changes in this PR return the gdal result which is not identical to the test data.